torchtext/data/iterator.py

```diff
@@ -286,8 +284,6 @@ def pool(data, batch_size, key, batch_size_fn=lambda new, count, sofar: count,
             if sort_within_batch \
             else batch(p, batch_size, batch_size_fn)
         if shuffle:
-            for b in random_shuffler(list(p_batch)):
```

Comment:
Is this relevant to the drop of python2 support?

Reply:
Yes, it's using Python 3 syntax that was not possible when also supporting Python 2.